### PR TITLE
Localization: Localized error message for server connection issue

### PR DESCRIPTION
**Pull request by josericardo-fo**
## What are the changes?

Localization and translation

## de
image

## fr
image

## ko
image

## pt_br
image

## zh_cn
image

## How to test the changes?

https://i.ember.zone/jac3ZDJoiy2b7zfV.png
Add this to start() in LoginPhase
```ts
// ...
start(): void {
    super.start();
    this.scene.unshiftPhase(new UnavailablePhase(this.scene));
    super.end();
    return;
// ...
```
## Checklist
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?
- [X] Have I tested the changes (manually)?
    - [X] Are all unit tests still passing? (`npm run test`)
- [X] Are the changes visual?
  - [X] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/de/menu.ts [hunk 1]
```diff
@@ -54,4 +54,5 @@ export const menu: SimpleTranslationEntries = {
   "disclaimer": "HAFTUNGSAUSSCHLUSS",
   "disclaimerDescription": "Dieses Spiel ist ein unfertiges Produkt. Es kann spielbeinträchtigende Fehler (bis hin zum Verlust des Speicherstandes)\n aufweisen, sich ohne Vorankündigung ändern und es gibt keine Garantie dass es weiterentwickelt oder fertiggestellt wird.",
   "choosePokemon": "Choose a Pokémon.",
+  "errorServerDown": "Oops! There was an issue contacting the server.\n\nYou may leave this window open,\nthe game will automatically reconnect.",
```
Comment (CodeTappert) [inline src/locales/de/menu.ts:57]:
```suggestion
  "errorServerDown": "Ups! Es gab einen Fehler beim Versuch den Server zu kontaktieren\nLasse dieses Fenster offen\nDu wirst automatisch neu verbunden.",
```

File: src/locales/fr/menu.ts [hunk 2]
```diff
@@ -49,4 +49,5 @@ export const menu: SimpleTranslationEntries = {
   "disclaimer": "AVERTISSEMENT",
   "disclaimerDescription": "Ce jeu n’est pas un produit fini et peut contenir des problèmes de jouabilité, dont de possibles pertes de sauvegardes,\ndes modifications sans avertissement et pourrait ou non encore être mis à jour ou terminé.",
   "choosePokemon": "Sélectionnez un Pokémon.",
+  "errorServerDown": "Oops! There was an issue contacting the server.\n\nYou may leave this window open,\nthe game will automatically reconnect.",
```
Comment (Adri1) [inline src/locales/fr/menu.ts:52]:
```suggestion
  "errorServerDown": "Oupsi ! Un problème de connexion au serveur est survenu.\n\nVous pouvez garder cette fenêtre ouverte,\nle jeu se reconnectera automatiquement.",
```

**Review by Adri1 — COMMENTED** [review]
Added French

**Review by Adri1 — APPROVED** [review]

File: src/locales/ko/menu.ts [hunk 3]
```diff
@@ -54,4 +54,5 @@ export const menu: SimpleTranslationEntries = {
   "disclaimer": "면책 조항",
   "disclaimerDescription": "이 게임은 완전히 개발되지 않았습니다- (세이브 데이터 소실을 포함) 플레이에 지장을 주는 문제가 생길 수 있으며,\n공지 없이 업데이트가 진행 혹은 중지될 수 있습니다.",
   "choosePokemon": "포켓몬을 선택하세요.",
+  "errorServerDown": "Oops! There was an issue contacting the server.\n\nYou may leave this window open,\nthe game will automatically reconnect.",
```
Comment (EnochG1) [inline src/locales/ko/menu.ts:57]:
```suggestion
  "errorServerDown": "서버 연결 중 문제가 발생했습니다.\n\n이 창을 종료하지 않고 두면,\n게임은 자동으로 재접속됩니다.",
```

**Review by EnochG1 — COMMENTED** [review]
Korean Translation

File: src/locales/zh_CN/menu.ts [hunk 4]
```diff
@@ -54,4 +54,5 @@ export const menu: SimpleTranslationEntries = {
   "disclaimer": "免责声明",
   "disclaimerDescription": "这个游戏尚未完成; 可能存在游戏性问题（包括潜在的丢档风险）、\n 不经通知的调整、 未来可能会更新或完成更多内容",
   "choosePokemon": "选择一只宝可梦。",
+  "errorServerDown": "Oops! There was an issue contacting the server.\n\nYou may leave this window open,\nthe game will automatically reconnect.",
```
Comment (40chyan) [inline src/locales/zh_CN/menu.ts:57]:
```suggestion
  "errorServerDown": "糟糕！访问服务器时发生了错误。\n\n你可以保持页面开启，\n游戏会自动重新连接。",
```
cn translation

**Comment by josericardo-fo** [comment]
@CodeTappert The german translation is too big.

image

**Comment by CodeTappert** [comment]
> @CodeTappert The german translation is too big.
> 
> image
> 

Then just add a \n in the first sentence somewhere

**Comment by josericardo-fo** [comment]
> > @CodeTappert The german translation is too big.
> > image
> 
> Then just add a \n in the first sentence somewhere

fixed

**Review by Tempo-anon — APPROVED** [review]

**Pull request merged**